src/plugins/discover/public/application/angular/discover.js

```diff
@@ -169,10 +169,10 @@ function discoverController($route, $scope, Promise) {
   let inspectorRequest;
   let isChangingIndexPattern = false;
   const savedSearch = $route.current.locals.savedObjects.savedSearch;
-  $scope.searchSource = savedSearch.searchSource;
+  $scope.persistentSearchSource = savedSearch.searchSource;
```

Comment:
It seems to me this is not used anywhere outside the current scope? Could we already not store that on `$scope` and make it a local variable instead?

Reply:
Good point, fixed